nodejs/FatBeaconPeripheral/FatBeacon.js

```diff
@@ -0,0 +1,45 @@
+var bleno = require('bleno');
+var eddystoneBeacon = require('eddystone-beacon');
+
+/***********************Altering Library Functions**************************/
+/**
+ * In order to broadcast in the fatbeacon format, we override several
+ * functions in the advertisement-data module of the eddystone-beacon library.
+ */
+var AdvertisementData = 
+        require('eddystone-beacon/lib/util/advertisement-data');
+var Eir = require('eddystone-beacon/lib/util/eir');
+
+var FAT_BEACON_FRAME_TYPE = 0x0e;
+var MAX_URL_LENGTH = 18;
+var SERVICE_UUID = 'feaa';
+
+AdvertisementData.makeEirData = function(serviceData) {
```

Comment:
Could you add comments for the overridden library functions?  It's hard to understand why these are needed for fatbeacon.